configs/full_configs/mlm_baseline.yaml

```diff
@@ -0,0 +1,75 @@
+model:
+  core_model:
+    core_model_type: generic
+    num_layers: 10
+    ffn:
+      ffn_type: swiglu
+      ffn_dim: 1536
+      normalization: rms_norm
+      bias: false
+    attn:
+      attn_type: generic
+      num_heads: 16
+      normalization: rms_norm
+      group_size: 4
+      bias: false
+      is_causal: false
+  embedder:
+    tokenizer_type: gpt2
+    embedding_model_type: generic
+    dataset_name: simple_en_wiki
```

Comment:
change this to stlm

Reply:
sorry, can I clarify what should be changed to stlm?

Reply:
the dataset_name, but its okay, this will all be reworked later